aws-attach-roles/README.md

```diff
@@ -25,6 +25,39 @@ YOUR-CLUSTER-NAME="<your-eks-cluster-name>"
 
 `aws s3 mb s3://${AWS_object_store_bucket} --region `
 
+- If your 2nd cluster running on different AWS accounts, you need to set approriate permission and IAM policy to allow Thanos sidecar put data on central S3 bucket. You can read more on how to do it in official AWS documentation [here](https://aws.amazon.com/premiumsupport/knowledge-center/cross-account-access-s3/)
```

Comment:
Are we suggesting following both of these bullet points? Or one or the other?

Reply:
Hi Sean, we recommend using 'S3 bucket policy' for POC or standard deployment and we recommend to use 'AWS Cross-Account AssumeRole' for advanced use case or when customer want to apply further restricted policy. Thank you for the feedback, I will update more clarification before merging this PR.